app/models/solidus_stripe/gateway.rb

```diff
@@ -34,12 +34,15 @@ def initialize(options)
     # We create and confirm the Stripe payment intent in two steps. That's to
     # guarantee that we associate a Solidus payment on the creation, and we can
     # fetch it after the webhook event is published on confirmation.
+    #
+    # The Stripe payment intent id is copied over the Solidus payment
+    # `response_code` field.
     def authorize(amount_in_cents, source, options = {})
       check_given_amount_matches_payment_intent(amount_in_cents, options)
 
       stripe_payment_intent = create_confirmed_stripe_payment_intent(
         source,
-        options[:originator].order,
+        options[:originator],
```

Comment:
nit: I was thinking that maybe would be clearer what we're expecting if it was all keyword arguments, would that work for you?

Reply:
I agree it's clearer. Updated.